SSA/Projects/InstCombine/Tactic.lean

```diff
@@ -1,4 +1,3 @@
--- import SSA.Core.WellTypedFramework
```

Comment:
Should this not import `SSA.Core.Framework` instead?

Reply:
It gets transitively imported anyway, so the import has no effect (but might still be desirable for explicitness, I'll leave that for your judgement).

It got commented because for some reason importing both `TSSA` and the new framework caused an error, but now that reason is gone so it's fine to uncomment it.

Anyway, not a blocker, feel free to merge